src/sympc/grads/grad_functions.py

```diff
@@ -135,6 +135,12 @@ def forward(ctx: Dict[str, Any], x: MPCTensor, y: Any) -> MPCTensor:
         Returns:
             x + y (MPCTensor): The result of the addition
         """
+        if isinstance(x, (list, int, float)):
```

Comment:
MAYBE: I think you should remove this because it reached this point because x was an MPCTensor, right?